mta/entities/applications.py

```diff
@@ -24,10 +24,9 @@
 class ApplicationsView(BaseLoggedInPage):
 
     title = Text(locator=".//div/h2[normalize-space(.)='Applications']")
-    add_application_button = Button("Add")
-    upload_file = HiddenFileInput(id="fileUpload")
-    search = Input("searchValue")
-    close_search = Text(locator=".//span[@class='pficon pficon-close']")
+    add_application_button = Text(locator='.//button[contains(text(), "Add application")]')
```

Comment:
Changing button Import to patternfly 4 will fix the problem .

     from widgetastic_patternfly4 import Button
     add_application_button = Button("Add application")

Reply:
fixed!